README.md

```diff
@@ -199,6 +199,11 @@ Ice server, and you can call services hosted by an IceRPC server from an Ice cli
 
 [IceRPC for Ice users] provides all the details.
 
+## Telemetry
+
+IceRPC collects completely anonymous telemetry data during the build process. Participation in this anonymous program is optional, and you may opt-out if you do not wish to share any information. Detailed information about the telemetry
+data collected and how to opt-out can be found in the [Telemetry] README.
```

Comment:
1) This is way over the 120 character limit.
2) To me, it feels like it's over-harping on how anonymous we are: "completely anonymous" "this anonymous 
program" I'd just mention it once at the beginning and leave it at that
3) "information about the telemetry data collected" is definitely correct English, but idk, it sounds a little weird to me.

Just to demonstrate (because you'd need to fix the line length anyways):
```suggestion
IceRPC collects anonymous telemetry data during the build process. Participation in this program is optional, and you may opt-out if you do not wish to share any information. Detailed information about what data is
collected and how to opt-out can be found in the [Telemetry] README.
```